packages/commerce-sdk-react/src/components/ShopperExperience/Component/index.test.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {render} from '@testing-library/react'
+import Component from './index'
+import {PageContext} from '../Page'
+
+const SAMPLE_COMPONENT = {
+    id: 'rfdvj4ojtltljw3',
+    typeId: 'commerce_assets.carousel',
+    data: {
+        title: 'Topseller',
+        category: 'topseller'
+    },
+    regions: [
+        {
+            id: 'regionB1',
+            components: [
+                {
+                    id: 'rfdvj4ojtltljw3',
+                    typeId: 'commerce_assets.carousel',
+                    data: {
+                        title: 'Topseller',
+                        category: 'topseller'
+                    }
+                }
+            ]
+        }
+    ]
+}
+
+const TEST_COMPONENTS = {
+    ['commerce_assets.carousel']: () => <div className="carousel">Carousel</div>
+}
+
+test('Page throws if used outside of a Page component', () => {
+    expect(() => render(<Component component={SAMPLE_COMPONENT} />)).toThrow()
+})
+
+test('Page renders correct component', () => {
```

Comment:
I see both test passing but I noticed a console error:
```
console.error
    Error: Uncaught [Error: "usePageContext" cannot be used outside of a page component.]
```

Is that expected?

**STR**
```
npm run test src/components/ShopperExperience/Component/index.test.tsx
```

Note: Remove these lines in jest config if you want to see more context.
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/ff97b2555170145ad4b10ac8ce05f1569293c71b/packages/commerce-sdk-react/jest.config.js#L24-L27